### PR TITLE
Add 64-bit iOS SEP xtr plugin ##bin

### PR DESCRIPTION
**Pull request by mrmacete**
Supports only 64-bit SEP firmwares.

It's based on a single sample, i'm glad to make it more generic if someone has more samples to share.

test with `rabin2 -x sep-firmware.n841.RELEASE.im4p-dec`

### CODE REVIEW

File: libr/bin/p/bin_xtr_sep64.c [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+    ut64 size_data;
+    ut64 virt;
+    ut64 entry;
+    ut64 unknown4;
+    ut64 unknown5;
+    ut64 unknown6;
+    ut32 minus_one;
+    ut32 unknown7;
+    char app_name[16];
+    ut8 app_uuid[16];
```
Comment (radare) [inline libr/bin/p/bin_xtr_sep64.c]: Use tabs

File: libr/bin/p/bin_xtr_sep64.c [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+		}
+		return res;
+	}
+	return NULL;
+}
+
+static RSepXtr64Ctx * sep64_xtr_ctx_new(RBuffer * buf) {
+	RSepHdr64 * hdr = NULL;
+	RSepApp64 * apps = NULL;
+	RSepXtr64Ctx * ctx = NULL;
```
Comment (radare) [inline libr/bin/p/bin_xtr_sep64.c]: Define and assign in the same line below

File: libr/bin/p/bin_xtr_sep64.c [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+beach:
+	free (hdr);
+	free (apps);
+	free (ctx);
+
+	return NULL;
+}
+
+static void sep64_xtr_ctx_free(void * p) {
+	if (p == NULL) {
```
Comment (radare) [inline libr/bin/p/bin_xtr_sep64.c]: Use !p instead if ==NULL

File: libr/bin/p/bin_xtr_sep64.c (hunk 2 above)
Comment (mrmacete) [reply libr/bin/p/bin_xtr_sep64.c]: hmm moving the declarations down will complicate the `goto beach;` "pattern"

**Pull request merged**